### PR TITLE
36 inventory argument

### PR DESCRIPTION
**Pull request by terryg**
Added ANSIBLE_INVENTORY to work like HERBERT_BRANCH.

### CODE REVIEW

File: ci/ansible/kiosk.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+- hosts: kiosks
+
+  vars:
+    - service_description: 'Herbert kiosk service'
+    - service_user: root
+    - service_file: dist/worker/main.js
```
Comment (robgonnella) [inline ci/ansible/kiosk.yml:6]: Should this be `"{{ appDir }}/scripts/kiosk.sh"` ?

File: ci/ansible/kiosk.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        path: '{{ appDir }}'
+        state: directory
+
+    - name: Sync Herbert scripts
+      synchronize:
+        src: '{{ srcDir }}/scripts'
+        dest: '{{ appDir }}'
+        recursive: yes
+        delete: yes
+
```
Comment (robgonnella) [inline ci/ansible/kiosk.yml:32]: I think to install and start the service we need to add

```yml
    - name: Install kiosk service
      become: yes
      template:
        src: ./templates/systemd-service.j2
        dest: /etc/systemd/system/herbert-kiosk.service
        owner: root
        group: root
        mode: '0644'

    - name: Enable and start herbert-kiosk service
      become: yes
      systemd:
        name: herbert-kiosk
        state: restarted
        daemon_reload: yes
        enabled: yes
```

File: ci/Makefile [hunk 3]
```diff
@@ -11,16 +11,26 @@ ifeq ($(HERBERT_BRANCH),)
...
+.PHONY: check_inventory
+check_inventory:
+ifeq ($(ANSIBLE_INVENTORY),)
+	@echo ""
+	@echo "Must set ANSIBLE_INVENTORY. Use \"ansible/inventory\" to use invetory"
+  @echo	"from current repository."
+	@echo ""
+	@exit 1
+endif
+
```
Comment (robgonnella) [inline ci/Makefile:23]: Question: Should we default `ANSIBLE_INVENTORY` to `ansible/inventory` if not set? Or is it better to force the user to always set this var?

If we want to default it we would just add the following to the top of the Makefile

```Makefile
ANSIBLE_INVENTORY ?= ansible/inventory
```

File: ci/ansible/kiosk.yml (hunk 1 above)
Comment (terryg) [reply ci/ansible/kiosk.yml:6]: I did not mean to include this!

File: ci/ansible/kiosk.yml (hunk 2 above)
Comment (terryg) [reply ci/ansible/kiosk.yml:32]: I didn't mean to include this. The kiosk starts when the user logs in, via .bashrc calling xinit. Not sure how to add that here. Will move this out of the pull request tho.

File: ci/Makefile (hunk 3 above)
Comment (terryg) [reply ci/Makefile:23]: Uh huh. Likewise add "HERBERT_BRANCH ?= local" or should the branch be required?

Comment (robgonnella) [reply ci/Makefile:23]: Sure I think that's a good default too!

**Comment by robgonnella** [comment]
@terryg just checking in, any reason not to merge this? It looks good to me

**Comment by terryg** [comment]
I wasn't allowed to merge at one point, and then forgot about it! I found a workaround too, which might be better. Symbolic link to an inventory file. ??

**Pull request closed without merging**